tools/cli/workflow.go

```diff
@@ -134,6 +134,17 @@ func newWorkflowCommands() []cli.Command {
 					Value: defaultDecisionTimeoutInSeconds,
 					Usage: "Decision task start to close timeout in seconds",
 				},
+				cli.StringFlag{
```

Comment:
Could you also add this to `RunWorkflow`? There are duplicate code from RunWorkflow and `StartWorkflow` which would be good to refactor.